src/tests/suites/ExpectedCLIVersion-Test.ts

```diff
@@ -0,0 +1,16 @@
+import { assert, expect } from 'chai';
```

Comment:
We already have a couple of unit tests, such as: https://github.com/Azure/vscode-bridge-to-kubernetes/blob/main/src/tests/suites/KubectlClient-Test.ts
Would it make sense to either transition them all to use Chai, or to move this test instead to the framework we already use?

Reply:
Done. ❤️☕️🎊 Awesome, definitely sounds good to me.